nilearn/plotting/js_plotting_utils.py

```diff
@@ -167,14 +167,19 @@ def open_in_browser(self, file_name=None, temp_file_lifetime=30):
         if file_name is None:
             fd, file_name = tempfile.mkstemp('.html', 'nilearn_surface_plot_')
```

Comment:
Turn this into a context

Reply:
no, the file must still exist after the function returns. besides this is outside the scope of this pr

Reply:
Ok